sentences/sv/_common.yaml

```diff
@@ -59,10 +59,10 @@ expansion_rules:
   är: "(är det | är)"
   brightness: "{brightness} [percent]"
   slå_på: "(tänd | sätt (på | igång | fart på) | slå (på | till) | starta)"
-  slå_av: "(slå (av | från | ifrån) | släck | stäng av | stoppa)"
+  slå_av: "(slå (av | från | ifrån) | släck | stäng (av) | stoppa)"
   i_på: "(i | på | vid | bredvid)"
   ändra: "(ställ in | sätt | justera | ändra | vrid (upp | ner))"
-  temp: "(temperatur)"
+  temp: "(temperatur[en])"
```

Comment:
I don't think this is used, in that case we should add a test case. It's a good idea to separate different intents/changes into separate PR.

Reply:
I'll remove this for now then. If seen needed we must add it + the test cases later.